hedera-node/src/main/java/com/hedera/services/ServicesState.java

```diff
@@ -237,6 +235,11 @@ public void init(Platform platform, AddressBook addressBook) {
 			setChild(ChildIndices.TOKEN_ASSOCIATIONS, new FCMap<>());
 			setChild(ChildIndices.DISK_FS, new MerkleDiskFs());
 			setChild(ChildIndices.SCHEDULE_TXS, new FCMap<>());
+
+			/* Initialize the running hash leaf at genesis to an empty hash. */
+			final var firstRunningHash = new RunningHash();
+			firstRunningHash.setHash(emptyHash);
+			setChild(ChildIndices.RECORD_STREAM_RUNNING_HASH, new RecordsRunningHashLeaf(firstRunningHash));
```

Comment:
Is this section for migration from older version? Is so, will the initialization to empty hash cause some information loss?